src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/pages/HmrcYesNoQuestionPage.scala.html

```diff
@@ -0,0 +1,78 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.hmrcfrontend.views.html.helpers.HmrcStandardPage
+
+@this(
+  hmrcStandardPage: HmrcStandardPage,
+  formWithCSRF: FormWithCSRF,
+  govukErrorSummary: GovukErrorSummary,
+  govukRadios: GovukRadios,
+  govukButton: GovukButton
+)
+
+@(questionMessage: String, form: Form[_], action: Call, pageTitle: Option[String] = None, name: Option[String] = None, backLink: Option[BackLink] = None, submitMessage: Option[String] = None)(implicit request: Request[_], messages: Messages)
+
+@hmrcStandardPage(
+  pageTitle = pageTitle,
+  backLink = backLink
+) {
+  @defining(questionMessage.replaceAll("\\.", "-")) { kebabCaseQuestion =>
```

Comment:
this would probably a common helper, and this is probably a naive solution - basically converting a message key into an id-like string, like `child-benefit.question.did-you-receive` to `child-benefit-question-did-you-receive`